### PR TITLE
Test/e2e speed up

### PR DESCRIPTION
**Pull request by yrong**

### CODE REVIEW

**Comment by yrong** [comment]
- [x] split bootstrap test to two directions as to be parallelized
- [x] hack lodestar param

with this hack we can decrease execution time of bootstrap test by at least 50%

**Review by claravanstaden — APPROVED** [review]
🤩 Incredible!! 👏 This will save us so much time, thank you @yrong! I tested locally and it works perfectly. I am surprised that the Bootstrap tests can be run in parallel because I know the Substrate -> Eth one needed to be run first to set up SnowETH to transfer in the first place, but looks like it works fine. 🤷‍♀️ 

I made a small PR to your PR to print out the time start-services took and also to remove an unnecessary check for the beacon relayer that speeds up the script a little. https://github.com/Snowfork/snowbridge/pull/750

**Comment by claravanstaden** [comment]
PS. The Lodestar `SLOTS_PER_EPOCH` is brilliant! 😂 Works like a charm.

File: core/packages/test/src/subclient/index.js [hunk 1]
```diff
@@ -13,6 +13,23 @@ class SubClient {
     const provider = new WsProvider(this.endpoint);
     this.api = await ApiPromise.create({
       provider,
+      rpc: {
+        basicOutboundChannel: {
+          getMerkleProof: {
```
Comment (doubledup) [inline core/packages/test/src/subclient/index.js:18]: Thanks for this! That warning was annoying

**Review by doubledup — APPROVED** [review]
This looks great! 🚀

**Review by vgeddes — APPROVED** [review]
+1

**Pull request merged**